src/v3/src/img/yubikeyDemo.svg

```diff
@@ -1,6 +1,6 @@
-<title id="yubikeyDemo">yubikey</title>
-<path fill-rule="evenodd" clip-rule="evenodd" d="M11.6199 62.9308C8.10518 59.416 8.10518 53.7176 11.6199 50.2028L14.4492 47.3735H17.2776L13.0341 51.617C10.3004 54.3507 10.3004 58.7829 13.0341 61.5165L30.7118 79.1942C33.4454 81.9279 37.8776 81.9279 40.6113 79.1942L82.5663 37.2392L76.0255 30.6985L61.5299 16.2028L54.9891 9.66204L25.7068 38.9443L24.2926 37.5301L54.282 7.54072L54.9891 6.83362L55.6962 7.54072L62.237 14.0815L68.7777 7.54072L69.4848 6.83362L70.1919 7.54072L84.6876 22.0364L85.3947 22.7435L84.6876 23.4506L78.1469 29.9914L84.6876 36.5321L85.3947 37.2392L84.6876 37.9463L42.0255 80.6084C38.5108 84.1231 32.8123 84.1231 29.2976 80.6084L11.6199 62.9308ZM76.7326 28.5772L82.5663 22.7435L69.4848 9.66204L63.6512 15.4957L76.7326 28.5772ZM54.1278 37.6716C58.033 41.5769 58.033 47.9085 54.1278 51.8137C50.2225 55.719 43.8909 55.719 39.9856 51.8137C36.0804 47.9085 36.0804 41.5769 39.9856 37.6716C43.8909 33.7664 50.2225 33.7664 54.1278 37.6716ZM55.542 36.2574C60.2283 40.9437 60.2283 48.5417 55.542 53.228C50.8557 57.9142 43.2577 57.9142 38.5714 53.228C33.8851 48.5417 33.8851 40.9437 38.5714 36.2574C43.2577 31.5711 50.8557 31.5711 55.542 36.2574Z" fill="#00297A" class="siwIconFillPrimaryDark"/>
-<path fill-rule="evenodd" clip-rule="evenodd" d="M23.2759 45.3735L17.9067 50.7427L19.6767 52.5027L28.0567 44.1227L19.6767 35.7427L17.9067 37.5027L23.2776 42.8735H0.0371857V45.3735H23.2759Z" fill="#A7B5EC" class="siwIconFillSecondary"/>
+<svg xmlns="http://www.w3.org/2000/svg" width="43" height="58" fill="none" viewBox="0 0 43 58">
+    <!-- TODO: title needs to be i18n/l10n-friendly -->
+    <title id="yubikeyDemo">yubikey</title>
+    <path fill="#00297A" fill-rule="evenodd" d="M10.61 0a9 9 0 0 0-9 9v.5a8.5 8.5 0 0 0 8.5 8.5v-2a6.5 6.5 0 0 1-6.5-6.5V9a7 7 0 0 1 7-7h13v14h-5.75v2h7.75v-3h8V3h-8V0h-15Zm15 5v8h6V5h-6Z" class="siwFillPrimary" clip-rule="evenodd"/>
+    <path fill="#A7B5EC" fill-rule="evenodd" d="M25.61 6.5h4v2h-4v-2Zm0 3h4v2h-4v-2Zm-15.356 2.486a2.986 2.986 0 0 1 5.973 0V32.86a1 1 0 1 0 2 0v-4.892a2 2 0 0 1 2-2h2.13a2 2 0 0 1 2 2v7.475a1 1 0 1 0 2 0v-1.047c0-1.633 1.352-2.975 3.005-2.975 1.619 0 2.944 1.315 2.944 2.914v5.764a1 1 0 1 0 2 0v-.614a3.042 3.042 0 1 1 6.084 0v4.683h2v-4.683a5.042 5.042 0 0 0-8.084-4.022v-1.128c0-2.724-2.24-4.914-4.944-4.914a5.003 5.003 0 0 0-3.006 1.002v-2.455a4 4 0 0 0-4-4h-2.13c-.728 0-1.411.195-2 .535V11.986a4.986 4.986 0 0 0-9.972 0v19.607c-1.398.5-2.847 1.495-4.072 2.492A29.391 29.391 0 0 0 .935 37.18l-.155.175-.062.226c-.146.542-.128 1.168-.044 1.783.086.633.254 1.345.478 2.1.447 1.509 1.139 3.276 1.93 5.082 1.581 3.616 3.6 7.483 4.977 9.943l.489.872 1.745-.977-.489-.873c-1.356-2.421-3.342-6.227-4.89-9.767-.776-1.772-1.431-3.453-1.845-4.849a12.468 12.468 0 0 1-.414-1.8 3.726 3.726 0 0 1-.038-.795 27.872 27.872 0 0 1 2.828-2.664c.943-.768 1.918-1.44 2.81-1.874v8.552a1 1 0 1 0 2 0V11.986Z" class="siwFillSecondary" clip-rule="evenodd"/>
```

Comment:
```suggestion
    <path fill="#A7B5EC" fill-rule="evenodd" d="M25.61 6.5h4v2h-4v-2Zm0 3h4v2h-4v-2Zm-15.356 2.486a2.986 2.986 0 0 1 5.973 0V32.86a1 1 0 1 0 2 0v-4.892a2 2 0 0 1 2-2h2.13a2 2 0 0 1 2 2v7.475a1 1 0 1 0 2 0v-1.047c0-1.633 1.352-2.975 3.005-2.975 1.619 0 2.944 1.315 2.944 2.914v5.764a1 1 0 1 0 2 0v-.614a3.042 3.042 0 1 1 6.084 0v4.683h2v-4.683a5.042 5.042 0 0 0-8.084-4.022v-1.128c0-2.724-2.24-4.914-4.944-4.914a5.003 5.003 0 0 0-3.006 1.002v-2.455a4 4 0 0 0-4-4h-2.13c-.728 0-1.411.195-2 .535V11.986a4.986 4.986 0 0 0-9.972 0v19.607c-1.398.5-2.847 1.495-4.072 2.492A29.391 29.391 0 0 0 .935 37.18l-.155.175-.062.226c-.146.542-.128 1.168-.044 1.783.086.633.254 1.345.478 2.1.447 1.509 1.139 3.276 1.93 5.082 1.581 3.616 3.6 7.483 4.977 9.943l.489.872 1.745-.977-.489-.873c-1.356-2.421-3.342-6.227-4.89-9.767-.776-1.772-1.431-3.453-1.845-4.849a12.468 12.468 0 0 1-.414-1.8 3.726 3.726 0 0 1-.038-.795 27.872 27.872 0 0 1 2.828-2.664c.943-.768 1.918-1.44 2.81-1.874v8.552a1 1 0 1 0 2 0V11.986Z" class="siwIconFillSecondary" clip-rule="evenodd"/>
```

Reply:
The `ImageWithText` component uses the `siwIcon*` class names. The reason for this was because of collision with the authCoin (which contains the same global classname).